backend/service/aws/aws.go

```diff
@@ -60,22 +58,22 @@ func New(cfg *any.Any, logger *zap.Logger, scope tally.Scope) (service.Service,
 	}
 
 	for _, region := range ac.Regions {
-		regionCfg := aws.NewConfig().WithRegion(region)
-		regionCfg.Retryer = awsclient.DefaultRetryer{
-			NumMaxRetries: clientRetries,
-		}
-
-		awsSession, err := session.NewSession(regionCfg)
-		if err != nil {
-			return nil, err
-		}
+		regionCfg, _ := config.LoadDefaultConfig(context.TODO(),
+			config.WithRegion(region),
```

Comment:
Oh ok let me look more closely I'm not familiar.

Reply:
Should I go simple here with a basic http client?
```go
	awsHTTPClient := &http.Client{
		Timeout: time.Second * 30,
	}
```

Or utilize some of the transport options the v1 sessions had set the following for TLS https://github.com/aws/aws-sdk-go/blob/df6799b65ee6bd2203d6c3bcb2f68e43f6031c7d/aws/session/session.go#L589-L601

Reply:
basic should work fine, don't think we were using any custom TLS or transport options.

also i think you can ignore a global timeout since we're threading context? unless you wanted a per-try timeout to work with the retrier... if you can determine the default timeout value in v1 you can stick with that for now

Reply:
Wasn't able to find the v1 timeout, going to keep it simple for now and remove the timeout.